packages/chain-adapters/package.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "axios": "^0.26.0",
+    "bech32": "^2.0.0",
```

Comment:
This works locally but CI isn't happy with it since it's a new package:
```
➜  shapeshiftLib git:(fix_validateAddress) ✗ yarn type-check
yarn run v1.22.17
$ lerna run type-check
lerna notice cli v4.0.0
lerna info versioning independent
lerna info Executing command in 6 packages: "yarn run type-check"
lerna info run Ran npm script 'type-check' in '@shapeshiftoss/types' in 0.9s:
$ tsc --project ./tsconfig.json --noEmit
lerna info run Ran npm script 'type-check' in '@shapeshiftoss/caip' in 1.1s:
$ tsc --project ./tsconfig.json --noEmit
lerna info run Ran npm script 'type-check' in '@shapeshiftoss/market-service' in 1.2s:
$ tsc --project ./tsconfig.json --noEmit
lerna info run Ran npm script 'type-check' in '@shapeshiftoss/chain-adapters' in 1.7s:
$ tsc --project ./tsconfig.json --noEmit
lerna info run Ran npm script 'type-check' in '@shapeshiftoss/asset-service' in 1.7s:
$ tsc --project ./tsconfig.json --noEmit
lerna info run Ran npm script 'type-check' in '@shapeshiftoss/swapper' in 1.2s:
$ tsc --project ./tsconfig.json --noEmit
lerna success run Ran npm script 'type-check' in 6 packages in 5.0s:
lerna success - @shapeshiftoss/asset-service
lerna success - @shapeshiftoss/caip
lerna success - @shapeshiftoss/chain-adapters
lerna success - @shapeshiftoss/market-service
lerna success - @shapeshiftoss/swapper
lerna success - @shapeshiftoss/types
✨  Done in 5.44s.
➜  shapeshiftLib git:(fix_validateAddress) ✗
```